plugin/src/main/resources/template/licenses/cc0.html

```diff
@@ -0,0 +1,147 @@
+<div class="library">
+  <h1 class="title">${library.name}</h1>
+  <% print library.url.isEmpty() ? "" : """<p><a href="${library.url}">${library.url}</a></p>""" %>
+  <div class="license">
+    <h2>CC0 1.0 Universal</h2>
+    <p>${library.copyrightStatement}</p>
```

Comment:
Do not needed because CC0 is no copyright.
https://creativecommons.org/publicdomain/zero/1.0/